api/v1/api_v1_quotas.go

```diff
@@ -29,23 +28,22 @@ func GetIsiQuota(
 	client api.Client,
 	path string) (quota *IsiQuota, err error) {
 
-	// PAPI call: GET https://1.2.3.4:8080/platform/1/quota/quotas
-	// This will list out all quotas on the cluster
+	// PAPI call: GET https://1.2.3.4:8080/platform/1/quota/quotas?path=/path/to/volume
+	// This will list the quota by path on the cluster
 
 	var quotaResp isiQuotaListResp
-	err = client.Get(ctx, quotaPath, "", nil, nil, &quotaResp)
+	var pathWithQueryParam = quotaPath + "?path=" + path
```

Comment:
Will path ever be empty?

Reply:
No. It can't be. There are other checks in its caller function to ensure that.